e2e/deployers/discoveredapps.go

```diff
@@ -0,0 +1,89 @@
+// SPDX-FileCopyrightText: The RamenDR authors
+// SPDX-License-Identifier: Apache-2.0
+
+package deployers
+
+import (
+	"fmt"
+
+	"github.com/ramendr/ramen/e2e/util"
+	"github.com/ramendr/ramen/e2e/workloads"
+)
+
+type DiscoveredApps struct{}
+
+func (s DiscoveredApps) GetName() string {
+	return "Disapp"
```

Comment:
Better to name as DiscoveredApp to avoid confusion later.

Reply:
The reason to not use full name is, in this PR I changed workload name format from Deployment-0 to Deployment-cephfs/rbd to better show the storage type. 

But OCM will create manifestwork and some label with format like namespace + name + suffix, for example:
subscr-deploy-cephfs-busybox-subscr-deploy-cephfs-busybox-ns-mw  

if I still use full name, it will exceed 64 bytes limit and cause issue. So for deployers DiscoveredApps, I choose to use Disapp as the short name.